### PR TITLE
add impacted area square for hazard map legend

### PR DESCRIPTION
**Pull request by loganmcdonald-noaa**
## What does this PR do? 🛠️
Finishes #1355 by adding the small legend square. 

Screenshot 2024-07-11 at 1 20 03 PM

### CODE REVIEW

**Review by eric-gade — APPROVED** [review]
LGTM 👍

**Comment by partly-igor** [comment]
Colors and legend are looking great, the only issue I am noticing is that on mobile the map has gotten a little squished (see screenshots below). I am guessing that the legend is counting in the aspect-ratio calculation? We want the map itself to be 3:2 (width:height), in the PR it's closer to 2.2:1 

**PR**
Screenshot of hazard map on design

**beta**
Screenshot of hazard map on beta

File: web/themes/new_weather_theme/templates/block/block--weathergov-local-alerts.html.twig [hunk 1]
```diff
@@ -105,20 +105,14 @@
                 <li class="">{{ areaName }}</li>
               {% endfor %}
             </ul>
-          {% endif %}
+            {% endif %}
 
-          <div class="grid-container margin-top-2 padding-x-0 tablet:padding-x-2">
-            <div class="grid-container padding-0 wx-isolation-isolate grid-col-12">
```
Comment (partly-igor) [inline web/themes/new_weather_theme/templates/block/block--weathergov-local-alerts.html.twig:111]: We'll want to move the `wx-isolation-isolate` class from here and put it on the `wx-alert-map` element

**Review by partly-igor — COMMENTED** [review]
It's looking really good, the only thing I noticed  with the last update is that the map control floating/z-index issue was reverted because the isolation class was removed. After that I think 👍

**Pull request merged**